File: ViewModels/ViewModels/AlbumViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Web;

namespace ViewModels.ViewModels
{
    public class AlbumViewModel : BaseViewModel
    {
        public string ImgKey { get; set; }

        public string Name { get; set; }

        public UserViewModel User { get; set; }


        public DateTime UploadDate { get; set; }


        public List<SongViewModel> Songs { get; set; }

        public HttpPostedFileBase File { get; set; }

        public string ImageBase64 { get; set; }
    }
}

File: src/musicgo/src/shared/classes/AlbumViewModel.ts
import { UserViewModel } from './UserViewModel';
import { SongViewModel } from './SongViewModel';

export class AlbumViewModel{

    public Id: string = '00000000-0000-0000-0000-000000000000';
    
    public Name: string = "";

    public User : UserViewModel | null = new UserViewModel();

    public ImgKey : string ="";

    public UploadDate :Date = new Date();

    public ImageBase64: string | null | ArrayBuffer = '';

    public Songs: SongViewModel[] = [];
}



    